package.json

```diff
@@ -2,7 +2,7 @@
   "name": "wait-for",
   "version": "2.2.3",
   "scripts": {
-    "test": "./node_modules/.bin/bats wait-for.bats",
+    "test": "bats wait-for.bats",
     "test:docker": "docker build -t wait-for-test . && docker run --rm -it wait-for-test"
```

Comment:
Isn't this broken now?

- Build-cache will prevent at times that the tests will run
- Running the image doesn't run the test, so we just print the help-text without any reason

Reply:
> Build-cache will prevent at times that the tests will run

You're right, I'm going to fix this by disabling the cache.

> Running the image doesn't run the test, so we just print the help-text without any reason

You're right, given the build cache issue. I'm going to fix this and also change it to be more explicit that we're aiming at running the tests.

Reply:
I've changed the scripts to disable the build cache.